application/modules/invoices/models/Mdl_invoices.php

```diff
@@ -206,17 +209,13 @@ public function create($db_array = null, $include_invoice_tax_rates = true)
                 $this->db->insert('ip_invoice_tax_rates', $db_array);
             }
         }
-
-        if($invoice_group !== '0') {
-            $this->load->model('invoice_groups/mdl_invoice_groups');
-            $invgroup = $this->mdl_invoice_groups->where('invoice_group_id', $invoice_group)->get()->row();
-            if (preg_match("/sumex/i", $invgroup->invoice_group_name)) {
-                // If the Invoice Group includes "Sumex", make the invoice a Sumex one
-                $db_array = array(
-                    'sumex_invoice' => $invoice_id
-                );
-                $this->db->insert('ip_invoice_sumex', $db_array);
-            }
+        $invgroup = $this->mdl_invoice_groups->where('invoice_group_id', $invoice_group)->get()->row();
```

Comment:
Use clear variable names, revert to $invoice_groups, also change code formatting